packages/router/src/bindings/messaging/auctionRequest.ts

```diff
@@ -41,9 +42,19 @@ export const auctionRequestBinding = async (
     logger.error("No data in auction request", requestContext, methodContext, err);
     return;
   }
+
   // On every new auction broadcast, route to the new auction handler
   logger.info("Received auction request", requestContext, methodContext);
   const { bid, bidSignature } = await newAuction(data, requestContext);
+
+  // check if new auction happens in 5s or not
+  const lastAttemptTime = lastAuctionMap.get(`${bid.user}-${bid.router}`) as number;
```

Comment:
Doesn't need to be keyed by `user-router`, we should key it by `user-sendingAsset-sendingChain-receivingAsset-receivingChain. We want to avoid a user spamming auctions for a swap combo, but for different swaps it's okay.